app/experimenter/experiments/views.py

```diff
@@ -72,6 +73,39 @@ def get_owner_display_value(self):
         if user_id is not None:
             return str(get_user_model().objects.get(id=user_id))
 
+    def get_display_start_date_info(self):
+        experiment_date_field = self.data.get("experiment_date_field")
+
+        date_after = None
+        date_before = None
+        if experiment_date_field == "starts_between":
+            date_after = self.data.get("date_range_after")
+            date_before = self.data.get("date_range_before")
+            type = "starting"
+        elif experiment_date_field == "pauses_between":
+            date_after = self.data.get("date_range_after")
+            date_before = self.data.get("date_range_before")
+            type = "pausing"
+        elif experiment_date_field == "ends_between":
+            date_after = self.data.get("date_range_after")
+            date_before = self.data.get("date_range_before")
```

Comment:
I think you can extract the `date_after` and `date_before` assignments to above the if statement.

Reply:
yea definitely, super oops.